src/Spout/Writer/ODS/Manager/WorksheetManager.php

```diff
@@ -197,7 +197,8 @@ private function getCellXML(Cell $cell, $styleIndex, $numTimesValueRepeated)
 
             $data .= '</table:table-cell>';
         } elseif ($cell->isBoolean()) {
-            $data .= ' office:value-type="boolean" calcext:value-type="boolean" office:boolean-value="' . $cell->getValue() . '">';
+            $value = $cell->getValue() ? 'true' : 'false';
```

Comment:
can you add a comment with the link to the spec? https://docs.oasis-open.org/office/v1.2/os/OpenDocument-v1.2-os-part1.html#datatype-boolean

Reply:
done.